app/Listeners/NetworkData/FlushEndorsementCache.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Listeners\NetworkData;
+
+use App\Events\NetworkData\AtcSessionEnded;
+use App\Models\Atc\Endorsement;
+use App\Notifications\AtcSessionRecordedConfirmation;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Support\Facades\Cache;
+
+class FlushEndorsementCache implements ShouldQueue
+{
+    public function handle(AtcSessionEnded $event)
+    {
+        $user = $event->atcSession->account;
+        $endorsementIds = Endorsement::pluck('id');
```

Comment:
Can convert to each loop since pluck returns a collection from memory? Know it is a small thing but open to extension and all that

Reply:
Nope good idea